bundle-workflow/tests/tests_sign_workflow/test_signer.py

```diff
@@ -41,3 +41,17 @@ def test_signer_checks_out_tool(self, mock_execute, mock_signer):
         Signer()
         self.assertEqual(mock_execute.call_count, 2)
         mock_execute.assert_has_calls([call("./bootstrap"), call("rm config.cfg")])
+
+    @patch("sign_workflow.signer.GitRepository")
+    @patch("sign_workflow.signer.GitRepository.execute")
+    def test_signer_verify(self,mock_object,mock_execute):
+        signer = Signer()
+        signer.verify("/path/the-jar.jar.asc")
+        mock_execute.assert_has_calls([call().execute("gpg --verify-files /path/the-jar.jar.asc")])
+
+    @patch("sign_workflow.signer.GitRepository")
+    @patch("sign_workflow.signer.GitRepository.execute")
+    def test_signer_sign(self, mock_object, mock_execute):
```

Comment:
You don't need to patch both `GitRepository` and `GitRepository.execute`, that's already a mock and you can access it as `mock_repo.execute....`. 

Rename `mock_object` to `mock_repo`.